presidio-anonymizer/tests/integration/test_anonymize_engine.py

```diff
@@ -9,6 +9,28 @@
 from presidio_anonymizer.operators import AESCipher
 
 
+def test_given_name_and_phone_number_then_we_anonymize_correctly_2():
```

Comment:
Maybe a name reflecting the scenario better?

Reply:
lol! 😂  missed that tnx.

Reply:
Done!